currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -36,9 +36,11 @@ def _get_close_time(self):
         CET Time is UTC+2 in summer and UTC+1 in winter:
         (https://en.wikipedia.org/wiki/Central_European_Time).
         Block time must be set to UTC+0
+        Set time to 15, which is UTC hour that corresponds to
+        16:00 CET in winter and 17:00 CET in summer => ok with the ECB publication time
         """
         if self.service == "ECB":
-            return 19  # 18:30
```

Comment:
According to https://www.ecb.europa.eu/services/contacts/working-hours/html/index.en.html the correct time is 17:30, and in UTC+0 it would be 18:30, therefore 19 should be kept (as it was).

Reply:
@victoralmau No, you are wrong. Please read the comment, it has the correct URL

Reply:
Correct URL is : https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html
It is written in the comment of the code, so please read the comment !

Reply:
```
The reference rates are usually updated around 16:00 CET on every working day
```

Reply:
No. For the update of exchange rates, there is just one time: it's 16:00 CET ; it is very clear on the page https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html And, from my experience, it is really 16:00 CET in real life. So don't change it.